src/parameters/parameters_flow_solver.cpp

```diff
@@ -190,6 +196,12 @@ void FlowSolverParam::parse_parameters(dealii::ParameterHandler &prm)
         else if (flow_case_type_string == "gaussian_bump")              {flow_case_type = gaussian_bump;}
 
         poly_degree = prm.get_integer("poly_degree");
+        
+        // get max poly degree for adaptation
+        max_poly_degree_for_adaptation = prm.get_integer("max_poly_degree_for_adaptation");
+        // -- set value to poly_degree if it is the default value
+        if(max_poly_degree_for_adaptation == 0) max_poly_degree_for_adaptation = poly_degree;
```

Comment:
Gaussian bump cases need to set this explicitly when they update the `poly_degree` in the loop where the parameters object is being changed then passed to `create_FlowSolver`

Reply:
Addressed in most recent commit